relay-general/src/protocol/contexts.rs

```diff
@@ -371,6 +385,7 @@ pub struct TraceContext {
 
     /// Whether the trace failed or succeeded. Currently only used to indicate status of individual
     /// transactions.
+    #[metastructure(required = "true")]
```

Comment:
@untitaker I don't think this is required.

Reply:
Just saw this comment, went to check sourcecode and saw that this was already fixed (just in case I find this comment again at some point in the future)

Reply:
Yep. No problem. Thanks for fixing. 👍